### PR TITLE
Exchange issue tracker URL on PR template

### PR DESCRIPTION
**Pull request by NotMyFault**
Follow up to https://github.com/jenkinsci/jenkins/pull/6477

### CODE REVIEW

**Review by daniel-beck — APPROVED** [review]

**Review by MarkEWaite — APPROVED** [review]

**Comment by MarkEWaite** [comment]
This PR is now ready for merge.  We will merge it after approximately 24 hours if there is no negative feedback.

**Review by basil — APPROVED** [review]
> This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback.

Mark, the 24-hour waiting period for a change like this is just unnecessary. Invoking the _de minimis_ clause ([clause 4](https://github.com/jenkinsci/jenkins/blob/36fa157ff15b1e4999c54825ce479e3cdebd21fb/docs/MAINTAINERS.adoc#exceptions)).

**Pull request merged**
